pkg/operator/controller/ingress/controller.go

```diff
@@ -348,6 +356,11 @@ func tlsProfileSpecForSecurityProfile(profile *configv1.TLSSecurityProfile) *con
 			}
 			return &configv1.TLSProfileSpec{}
 		} else if spec, ok := configv1.TLSProfiles[profile.Type]; ok {
+			// TODO remove when haproxy is built with an openssl version that supports tls v1.3.
+			if profile.Type == configv1.TLSProfileModernType {
+				log.Info("converted modern tls security profile to intermediate")
```

Comment:
This log message doesn't have enough context and isn't actionable, maybe remove it entirely?